kernel/src/actions/mod.rs

```diff
@@ -602,7 +602,7 @@ impl CommitInfo {
             operation_parameters: None,
             kernel_version: Some(format!("v{KERNEL_VERSION}")),
             engine_info,
-            txn_id: None,
+            txn_id: Some(uuid::Uuid::new_v4().to_string()),
```

Comment:
maybe we could add a note to docstring above the field def that we just always write this